### PR TITLE
Readme update + text fixes

### PR DESCRIPTION
**Pull request by hubicios**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,8 +124,8 @@ Path for Hyperkvasir dataset (must contain folders `labeled-images` and `segment
 Path for ERS dataset (must contain patient id directories e.g. `0001`)  
 - `--ers-use-seq`  
 Use sequences directory for ERS dataset (e.g. "seq_01"). Defaults to false.
-- `--ers-use-empty-masks`  
-Flag specifying whether frames with empty mask files that are labeled as not healthy will be used for segmentation. Empty masks files that are healthy will be used nonetheless. See [healthy flag section](#healthy-flag). Defaults to false. For training type `classification` it is overriden to true.
+- `--ers-use-empty-masks` 
+Flag specifying whether images with empty mask files should be used for segmentation. Independently, script will use empty mask files that belong to healthy classes. See [healthy flag section](#healthy-flag). Defaults to false. For training type `multilabel-classification` it is overridden to true.
```
Comment (wrobcio789) [inline README.md]: The trick is really that all empty mask files, if processed, are created as an fully white image. We should add this information here. I think we can also remove the healthy flag section

File: README.md [hunk 2]
```diff
@@ -167,14 +160,14 @@ disease:
...
+Records assigned to class that are not mapped will be dropped, therefore will not exist in the output dataset. Default mapping behavior, that occurs when class mapper is unspecified, maps classes one to one, ex: `c01 -> c01` and assumes that `h01-h07` and `b02` are healthy classes.
 
-##### Healthy flag
-In ERS dataset, some masks are empty files, especially the ones that label healthy classes. By default, such mask files are ignored unless they belong to a class labeled as healthy. The script converts them to a valid picture which mode and size is based on the original frame. Filling color of the picture is determined by the following rule:
-- if the mask belongs to a healthy class **and** segmentation is not binary, the color is white.
-- otherwise, the color is black.
+##### Positive flag
+In ERS dataset, some masks are empty files, especially the ones that label healthy classes. By default, such mask files are ignored unless they belong to a class labeled as positive. The script converts them to a valid picture which mode and size is based on the original frame. Filling color of the picture is determined by the following rule:
```
Comment (wrobcio789) [inline README.md]: This positive flag now has nothing to do with the healthy or not healthy. It works only for binary segmentation and works as following:
1. If a class is postiive one, then masks that belong to it are merged 
2. If class is not positive, then the mask is going to be fully black

File: README.md [hunk 3]
```diff
@@ -167,14 +160,14 @@ disease:
...
+- if  segmentation is binary **and**, the mask belongs to a none positive class the color is black.
+- otherwise, the color is white.
```
Comment (wrobcio789) [inline README.md]: 167-168 to remove

File: README.md [hunk 4]
```diff
@@ -167,14 +160,14 @@ disease:
...
 
-If there is a valid mask file that belongs to a class labeled as healthy and the segmentation is not binary, then the colors in the mask will get reversed. White pixels will switch places with the black pixels.
+If there is a valid mask file that belongs to a class labeled as not positive, and the segmentation is binary, then the colors in the mask will get reversed. White pixels will switch places with the black pixels.
```
Comment (wrobcio789) [inline README.md]: No reversing, see comment above

File: README.md [hunk 5]
```diff
@@ -207,17 +200,14 @@ The example will be transformed to the following output:
...
-│   │	├── 1.png //Mask merging 00001_c01_c02_q01.png and 00001_c03_c04.png
-│   │	├── 2.png //Same as 00002_q02.png
+│   │	├── 00001.png //Mask merging 00001_c01_c02_q01.png and 00001_c03_c04.png
 │   ├── q_class
-│   │	├── 3.png //Same as 00001_c01_c02_q01.png
+│   │	├── 00001.png //Same as 00001_c01_c02_q01.png
+│   │	├── 00002.png //Same as 00002_q02.png
 ```
 
 ##### Mask merging
```
Comment (wrobcio789) [inline README.md:298]: In this section we should remove the `Reverse masks merging example` and add an example for non positive class in binary classification. That would be mask file going fully black

File: README.md [hunk 6]
```diff
@@ -167,14 +160,14 @@ disease:
...
-- otherwise, the color is black.
+##### Positive flag
+In ERS dataset, some masks are empty files, especially the ones that label healthy classes. By default, such mask files are ignored unless they belong to a class labeled as positive. The script converts them to a valid picture which mode and size is based on the original frame. Filling color of the picture is determined by the following rule:
+- if  segmentation is binary **and**, the mask belongs to a none positive class the color is black.
+- otherwise, the color is white.
 
-If there is a valid mask file that belongs to a class labeled as healthy and the segmentation is not binary, then the colors in the mask will get reversed. White pixels will switch places with the black pixels.
+If there is a valid mask file that belongs to a class labeled as not positive, and the segmentation is binary, then the colors in the mask will get reversed. White pixels will switch places with the black pixels.
 
 ##### Mapping example
```
Comment (wrobcio789) [inline README.md:174]: Section on output mask and frame naming would be needed. For ers it's `"patient_id}_{data_dir_basename}_{frame_name}.png"` and for hkvs we just copy the filename.

I think then this should be shown in mapping scheme too

**Review by wrobcio789 — CHANGES_REQUESTED** [review]

**Review by wrobcio789 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (hubicios) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (hubicios) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (hubicios) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (hubicios) [reply README.md:174]: Done

File: README.md (hunk 3 above)
Comment (hubicios) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (hubicios) [reply README.md:298]: Done

**Pull request merged**